go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/aws/aws-sdk-go v1.16.26
 	github.com/bazelbuild/bazel-gazelle v0.0.0-20181012220611-c728ce9f663e
 	github.com/bazelbuild/buildtools v0.0.0-20180226164855-80c7f0d45d7e
-	github.com/blang/semver v3.5.0+incompatible
+	github.com/blang/semver v3.5.1+incompatible
```

Comment:
bumped to the preferred version

Reply:
i think we should only be bumping if we need anything from the new version of these packages - semver, errors, testify

Reply:
if we don't bump these. we should downgrade the versions used in `k8s.io/system-validators. 

another solution is to bump these in other PRs then rebase this one. thoughts @neolit123 @rosti

Reply:
> if we don't bump these. we should downgrade the versions used in `k8s.io/system-validators.

hm, how did this happen, we just copied the validators there?
so both validators repo and k/k should have been the same.

Reply:
see https://github.com/kubernetes/system-validators/blob/master/go.mod#L7

I'm not sure, but maybe `go mod tidy` uses latest tag

Reply:
https://github.com/kubernetes/system-validators/pull/4

Reply:
should be fixed in the validators repo and a new tag is available:
https://github.com/kubernetes/system-validators/releases/tag/v1.0.1

Reply:
can we bump this in a different PR? this adds more dependencies to review as a dependency reviewer and this PR is already rather large. reviewing dep changes properly is time consuming.

Reply:
this was a mistake in the validators repository. the versions bumps should no longer be needed.

Reply:
I'll update the PR, the bumps shouldn't be needed